site/app/_components/PlayerHero.tsx

```diff
@@ -98,7 +107,7 @@ export default function PlayerHero(props: PlayerHeroProps) {
               )}
               style={{
                 aspectRatio: !theme.audio ? assetItem.aspectRatio : undefined,
-                width: theme.audio ? assetItem.aspectRatio * 719 : undefined,
```

Comment:
how can this be correct? when it's an audio theme this would set the width to the aspect ratio.